docs/ref/models/expressions.txt

```diff
@@ -560,13 +560,25 @@ calling the appropriate methods on the wrapped expression.
         nested expressions. ``F()`` objects, in particular, hold a reference
         to a column.
 
-    .. method:: asc()
+    .. method:: asc(**kwargs)
```

Comment:
This documentation seems to be in the wrong spot. This is under `Expression` not `F`. Should `BaseExpression` also have these changes or should the documentation be corrected?